src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -67,11 +67,6 @@ DlgPrefInterface::DlgPrefInterface(
     double unscaledDevicePixelRatio = m_dDevicePixelRatio / initialScaleFactor;
     m_minScaleFactor = 1 / unscaledDevicePixelRatio;
 
-    VERIFY_OR_DEBUG_ASSERT(m_pSkin != nullptr) {
```

Comment:
Have you made sure we don't assume that `m_pSkin != nullptr` anywhere else?

Reply:
yes